bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -233,6 +233,10 @@ class ReplicaConfig : public concord::serialize::SerializableFactory<ReplicaConf
                std::chrono::seconds,
                std::chrono::seconds{60},
                "Time interval in seconds to monitor disk usage by db checkpoints");
+  CONFIG_PARAM(dbCheckpointDiskSpaceThreshold,
+               std::string,
```

Comment:
Can we use a floating point type here?

Reply:
these config parameters need to be serializable and we do not have floating point number serialization support in current utility. So keeping it this way for now.